### PR TITLE
amqp-client: version 1.0.2

### PR DESCRIPTION
**Pull request by andersfugmann**

### CODE REVIEW

**Comment by andersfugmann** [comment]
If anyone has an idea on how to make amqp-client package be installed after both async and lwt. Currently a opam install lwt async amqp-client will only compile amqp-client against lwt, and hence needs to be installed separately after async / lwt.

File: packages/amqp-client/amqp-client.1.0.2/opam [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+bug-reports: "https://github.com/andersfugmann/amqp-client/issues"
+license: "BSD3"
+dev-repo: "https://github.com/andersfugmann/amqp-client.git"
+build: [ ]
+install: [make "install"]
+remove: [make "uninstall"]
+depends: [
+  "ocamlfind" {build}
+  "omake" {build}
+  "xml-light" {build}
```
Comment (avsm) [inline packages/amqp-client/amqp-client.1.0.2/opam]: is xml-light definitely a buildtime dependency only, or is it linked at runtime?

Comment (andersfugmann) [inline packages/amqp-client/amqp-client.1.0.2/opam]: Build time only. Its only used to parse the amqp specification when generating the autogenerated protocol description.

**Comment by andersfugmann** [comment]
Ready to merge. I have moved async and lwt to depopts, but the package is useless without any of these. However, travis does not test compilation against any depopts packages is seems.

File: packages/amqp-client/amqp-client.1.0.2/opam (hunk 1 above)
Comment (avsm) [inline packages/amqp-client/amqp-client.1.0.2/opam]: thanks for the clarification!

**Pull request merged**